{"uuid":"af872589-a93c-4e08-b72a-55265b97546b","historyId":null,"status":"broken","statusDetails":{"message":"Oops...we found an error preparing this test file:\n\n  > cypress\\e2e\\Features\\[04]LogInTelnyx.feature\n\n","trace":"\n\nError: Webpack Compilation Error\n./cypress/e2e/Features/[04]LogInTelnyx.feature 4:10\nModule parse failed: Unexpected token (4:10)\nYou may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders\n| \n|   Background:\n>     Given Login out if loginned in\n|     Given Im on home page\n|\n    at handle (C:\\Users\\snigu\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\@packages\\server\\node_modules\\@cypress\\webpack-preprocessor\\dist\\index.js:180:23)\n    at finalCallback (C:\\Users\\snigu\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:257:39)\n    at C:\\Users\\snigu\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:306:14\n    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\\Users\\snigu\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\HookCodeFactory.js:33:10), <anonymous>:6:1)\n    at AsyncSeriesHook.lazyCompileHook (C:\\Users\\snigu\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\Hook.js:154:20)\n    at C:\\Users\\snigu\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:304:22\n    at Compiler.emitRecords (C:\\Users\\snigu\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:499:39)\n    at C:\\Users\\snigu\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:298:10\n    at C:\\Users\\snigu\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:485:14\n    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\\Users\\snigu\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\HookCodeFactory.js:33:10), <anonymous>:6:1)\n    at AsyncSeriesHook.lazyCompileHook (C:\\Users\\snigu\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\Hook.js:154:20)\n    at C:\\Users\\snigu\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:482:27\n    at C:\\Users\\snigu\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\neo-async\\async.js:2818:7\n    at done (C:\\Users\\snigu\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\neo-async\\async.js:3522:9)\n    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\\Users\\snigu\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\HookCodeFactory.js:33:10), <anonymous>:6:1)\n    at AsyncSeriesHook.lazyCompileHook (C:\\Users\\snigu\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\Hook.js:154:20)\n    at C:\\Users\\snigu\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:464:33\n    at C:\\Users\\snigu\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\@packages\\server\\node_modules\\graceful-fs\\graceful-fs.js:143:16\n    at C:\\Users\\snigu\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\@packages\\server\\node_modules\\graceful-fs\\graceful-fs.js:143:16\n    at C:\\Users\\snigu\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\@packages\\server\\node_modules\\graceful-fs\\graceful-fs.js:61:14\n    at FSReqCallback.oncomplete (node:fs:188:23)\n\nThis occurred while Cypress was compiling and bundling your test code. This is usually caused by:\n\n- A missing file or dependency\n- A syntax error in the file or one of its dependencies\n\nFix the error in your code and re-run your tests."},"stage":"finished","attachments":[{"name":"video recording","type":"video/mp4","source":"1154885e-1f07-4d61-9115-6db91dd6e8d2-attachment.mp4"}],"parameters":[],"labels":[{"name":"suite","value":"cypress/e2e/Features/[04]LogInTelnyx.feature"}],"links":[],"start":1661362530949,"name":"Oops...we found an error preparing this test file: cypress/e2e/Features/[04]LogInTelnyx.feature","fullName":"Oops...we found an error preparing this test file: cypress/e2e/Features/[04]LogInTelnyx.feature","stop":1661362530949}